pages/profile/v2/index.tsx

```diff
@@ -60,7 +60,7 @@ export default function Profile() {
       </div>
       <div className="grid">
         <ExpandableContainer open={true} title={"Achievements"}>
-          <AchievementComponent user={user} />
+          <AchievementComponent userId={user.uid} isEditable={false} />
```

Comment:
set isEditable to be false so no one can edit badges. Next step is to set isEditable to be true if the logged in user is that user or a coach.